### PR TITLE
Release redis-0.22.0 / redis-test-0.1.0

### PR DESCRIPTION
**Pull request by jaymell**

### CODE REVIEW

File: redis/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,43 @@
...
+*   Add support for weights in zunionstore and zinterstore ([#641](https://github.com/redis-rs/redis-rs/pull/641))
+*   Cluster: Create read_from_replicas option ([#635](https://github.com/redis-rs/redis-rs/pull/635))
+*   Make Direction a public enum to use with Commands like `blmove` ([#646](https://github.com/redis-rs/redis-rs/pull/646))
+*   Add `ahash` feature for using ahash internally & for redis values
+*   Add Script::load function ([#603](https://github.com/redis-rs/redis-rs/pull/603))
+*   Add support for OBJECT ([[#610]](https://github.com/redis-rs/redis-rs/pull/610))
+*   Add GETEX and GETDEL support ([#582](https://github.com/redis-rs/redis-rs/pull/582))
+*   Add support for ZMPOP ([#605](https://github.com/redis-rs/redis-rs/pull/605))
+
+#### Breaking changes
```
Comment (djc) [inline redis/CHANGELOG.md:15]: I think we should have these first.

File: redis/CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,43 @@
+<a name="0.22.0"></a>
+### 0.22.0 (2022-09-30)
+
+#### Features
```
Comment (djc) [inline redis/CHANGELOG.md:4]: I like to also call out external contributors (that is, every PR author who is not you or me), how do you feel about that?

**Review by djc — COMMENTED** [review]
Thanks for doing the work to prepare this!

As for the changelog, I usually prefer putting it in GitHub Releases, which I think will make it more readily available in for example Dependabot PRs. Example from chrono:

https://github.com/chronotope/chrono/releases/tag/v0.4.20

Maybe also add a short introduction there to explain a bit of bigger picture?

File: redis/CHANGELOG.md (hunk 2 above)
Comment (jaymell) [reply redis/CHANGELOG.md]: Done.

**Comment by jaymell** [comment]
Comments addressed (hopefully to your satisfaction, let me know, or feel free to push anything else you'd like to see added).

I also fixed/updated the cargo release configs, though I'm not sure we actually need them at this point. 

I can also add proper releases in github, but I imagine we want to merge/publish before doing so?

**Comment by jaymell** [comment]
@djc  What are your thoughts about merging this? Any additional things you'd like to see?

**Review by djc — APPROVED** [review]

**Pull request merged**